cross/python3/patches/ppc853x/007-fix-endianness-detection.patch

```diff
@@ -0,0 +1,12 @@
+Fixes the endianness detection by forcing it to be big endian
+--- configure.ac
++++ configure.ac
+@@ -4734,7 +4734,7 @@ else
+ fi
+ 
+ # check for endianness
+-AC_C_BIGENDIAN
```

Comment:
Here the issue is that this macro actually *runs* compiled code to see the output.
Now, since we're running (in docker case at least) in a little endian environment it says... "little endian".

The test to validate the outcome is to run `python -c "import sys; print(sys.byteorder)`